scripts/products/get-setProducts-txdata.js

```diff
@@ -57,8 +176,8 @@ const main = async (provider, productsDataFilePath) => {
         (coverAssetsAsText === 'DAI' && 0b10) || // DAI only (2)
         (coverAssetsAsText === 'ETH' && 0b01) || // ETH only (1)
         (coverAssetsAsText === 'USDC' && 0b100000) || // USDC only (32)
-        (coverAssetsAsText === 'DAI,USDC' && 0b1000010) || // DAI & USDC (66)
-        (coverAssetsAsText === 'ETH,USDC' && 0b1000001) || // ETH & USDC (65)
+        ((coverAssetsAsText === 'DAI,USDC' || coverAssetsAsText === 'USDC,DAI') && 0b1000010) || // DAI & USDC (66)
```

Comment:
btw, one source of errors here could be if an accidental white space is introduced. then it'll default to 0